src/FlaUILibrary/flaui/module/element.py

```diff
@@ -82,110 +81,44 @@ def create_value_container(name=None, xpath=None, retries=None, use_exception=No
     def execute_action(self, action: Action, values: Container):
         """
         If action is not supported an ActionNotSupported error will be raised.
-

```

Comment:
@Nepitwin are you intentionally deleting these lines of comment or it is an accidental commit

Reply:
It's intentionally deleted all method can be read by lambdas so no need for an documentation. In future all moduls will be deleted by this lines.